sycl/doc/extensions/proposed/sycl_ext_oneapi_free_function_queries.asciidoc

```diff
@@ -16,71 +18,71 @@
 // docbook uses c++ and html5 uses cpp.
 :language: {basebackend@docbook:c++:cpp}
 
-== Introduction
-IMPORTANT: This specification is a draft.
-
-NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by permission by Khronos.
-
-This document describes an extension that adds features for SYCL work items and groups to be available globally.
 
 == Notice
 
-Copyright (c) 2020-2021 Intel Corporation.  All rights reserved.
-
-== Status
-
-Working Draft
-
-This is a preview extension specification, intended to provide early access to a feature for review and community feedback. When the feature matures, this specification may be released as a formal extension.
-
-Because the interfaces defined by this specification are not final and are subject to change they are not intended to be used by shipping software products.
+[%hardbreaks]
+Copyright (C) 2020-2024 Intel Corporation.  All rights reserved.
 
-== Version
+Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks
+of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by
+permission by Khronos.
 
-Revision: 1
 
 == Contact
-Ruslan Arutyunyan, Intel (ruslan 'dot' arutyunyan 'at' intel 'dot' com)
 
-John Pennycook, Intel (john 'dot' pennycook 'at' intel 'dot' com)
+To report problems with this extension, please open a new issue at:
 
-== Dependencies
+https://github.com/intel/llvm/issues
 
-This extension is written against the SYCL 2020 specification, Revision 4.
 
-== Feature Test Macro
+== Dependencies
 
-This extension provides a feature-test macro as described in the core SYCL
-specification section 6.3.3 "Feature test macros". Therefore, an implementation
-supporting this extension must predefine the macro `SYCL_EXT_ONEAPI_FREE_FUNCTION_QUERIES`
-to one of the values defined in the table below. Applications can test for the
-existence of this macro to determine if the implementation supports this
-feature, or applications can test the macro's value to determine which of the
-extension's APIs the implementation supports.
+This extension is written against the SYCL 2020 revision 8 specification.  All
+references below to the "core SYCL specification" or to section numbers in the
+SYCL specification refer to that revision.
```

Comment:
There should be a [Status](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/template.asciidoc#status) section here after Dependencies.  Use the suggested text under "Use this if the extension is a proposed new version of an existing extension".

Reply:
Oops.  Fixed in 49df15411e0e06f9c61c83cc7b91ac0a86655adc.